test/assembly/BUILD

```diff
@@ -22,9 +22,10 @@ java_library(
     srcs = [
```

Comment:
Why are we introducing the terminology `test/assembly`? Why not just:

```
test/
  Runner

  /server
    GraknCoreRunner
    GraknClusterRunner
    
  /console
    ConsoleRunner
```

Reply:
Because `assembly` is more consistent with how clients uses the term, but I can change it.

Reply:
Yeah, I think it's better to call it `server` and `console` since `assembly` is not a term that `common` should know about. In addition to that, `GraknClusterRunner` and `GraknCoreRunner` are not only used in `assembly`, but also in `test-behaviour` and also `test-reliability` (for `GraknClusterRunner`)